src/components/Common/Hr.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import is from 'styled-is'
+
+const Hr = styled.hr`
+  height: ${remcalc(1)};
+  margin: ${remcalc(17)} 0 ${remcalc(18)} 0;
+  border: none;
+  background-color: ${props => props.theme.colors.grey} ${is('short')`
+    width: 25%
+  `} ${is('muted')`
```

Comment:
```suggestion
  `};
   ${is('muted')`
```

Reply:
Thanks bud!!

I think this was an auto change on save in my text editor :rage: :roll_eyes: 

Good spot.